### ISSUE TITLE
PHP Notice:  Undefined offset: 0

### ISSUE THREAD

**Issue by ghost** [jdelamater99/Twitter-RSS-Parser#6]
If I am polling a list or search that has no results I see errors in the apache2 logs like this -

[Sun Sep 29 19:03:33 2013] [error] [client XXX.XXX.XXX.XXX] PHP Notice:  Undefined offset: 0 in /var/www/twitter-rss-parser/list.php on line 29, referer: https://URL/twitter-rss-parser/?list=LISTNAME&owner=USERNAME
[Sun Sep 29 19:22:40 2013] [error] [client XXX.XXX.XXX.XXX] PHP Notice:  Undefined offset: 0 in /var/www/twitter-rss-parser/search.php on line 30, referer: https://URL/twitter-rss-parser/?q=%23SEARCHQUERY

HTH

**Comment by vinzv** [comment]
Same here, on every call to the script.

**Comment by jdelamater99** [comment]
I've never experienced those errors. Can you give more detail on how to replicate them.

Also, I'm assuming that you've filled in the appropriate info in the top portion of the config.php-dist and renamed it to config.php

**Comment by ghost** [comment]
Yep , Twitter-RSS-Parser is running correctly , I think these errors occur when there are no results from the script.
